android-agent/src/main/kotlin/io/opentelemetry/android/agent/OtelRumConfigExtensions.kt

```diff
@@ -73,7 +75,7 @@ fun OtelRumConfig.addCrashAttributesExtractor(extractor: AttributesExtractor<Cra
     return this
 }
 
-fun OtelRumConfig.addNetworkChangeAttributesExtractor(extractor: AttributesExtractor<CurrentNetwork, Void>): OtelRumConfig {
+fun OtelRumConfig.addNetworkChangeAttributesExtractor(extractor: BiConsumer<AttributesBuilder, CurrentNetwork>): OtelRumConfig {
```

Comment:
Not to be addressed here, though I think it'd be nice to convert the underlying types to Kotlin (I'm referring to the NetworkChangeInstrumentation one in this case) to avoid having to propagate upward the usage of types that have better alternatives in Kotlin, such as `BiConsumer`, which can be a Kotlin function with 2 params.

Reply:
Yeah good point. I'll get that in a follow-up PR.